rules/enforce-store-naming-convention/enforce-store-naming-convention.md

```diff
@@ -1,11 +1,44 @@
 # effector/enforce-store-naming-convention
 
-Enforcing naming conventions helps keep the codebase consistent, and reduces overhead when thinking about how to name a variable with store. Your stores should be distingueshed by a prefix $. For example, `$name` is a store,`name` is not.
+Enforcing naming conventions helps keep the codebase consistent, and reduces overhead when thinking about how to name a variable with store. Depending on the configuration your stores should be distinguished by a prefix or a postfix $. Enforces prefix convention by default.
 
+## Prefix convention
+When configured as:
+```js
+module.exports = {
+  rules: {
+    "effector/enforce-store-naming-convention": "error",
+  },
+};
+```
+Prefix convention will be enforced:
 ```ts
 // 👍 nice name
 const $name = createStore(null);
 
 // 👎 bad name
 const name = createStrore(null);
 ```
+## Postfix convention
```

Comment:
Shall we move documentation about shred settings to main README? It seems like general info.

Reply:
I think yes it's a good idea to copy this info to README, but I'd also still keep the information here since it's related to the particular rule